frontend/.storybook/preview-head.html

```diff
@@ -2,3 +2,6 @@
   href="https://fonts.googleapis.com/css?family=Roboto:300,400,700"
   rel="stylesheet"
 />
+<script>
+  window.global = window;
+</script>
```

Comment:
Just curious, what does this solve?

Reply:
This was added automatically during the storybook vite migration, without it I'm getting this

<img width="939" alt="Screenshot 2023-02-27 at 09 46 23" src="https://user-images.githubusercontent.com/1403093/221516488-f4ad1867-78e4-4097-95dd-f964bf20e954.png">

I think (not very sure) it's some sort of polyfill for libraries that rely on `global.` instead of `window.`